src/reducers/pause.js

```diff
@@ -80,12 +87,7 @@ function update(state: PauseState = State(), action: Action): PauseState {
         frameScopes: { ...state.frameScopes, [selectedFrameId]: scopes }
       };
     case "RESUME":
-      return Object.assign({}, state, {
-        pause: null,
-        frames: null,
-        selectedFrameId: null,
-        loadedObjects: {}
-      });
+      return { ...state, emptyPauseState };
```

Comment:
Doesn't this create an object that looks like state, but with a new property named `emptyPauseState`?

Reply:
Yep, good catch. Turns out that we can drop this entirely now that we no longer update the state when we are resumed